app/assets/stylesheets/components/helpers/_variables.scss

```diff
@@ -10,8 +10,8 @@ $app-spacing-scale-6: 40px;
 $app-spacing-scale-7: 50px;
```

Comment:
Can we remove this spacing scale and replace uses with the govuk-spacing scale, seeing as this was forked from GOVUK  Frontend in the first place? It looks like it's only actually used in one place, so hopefully is easy to replace.

Reply:
Done, good spot, thanks.